src/main/resources/org/jenkinsci/plugins/workflow/support/steps/input/InputStep/help-submitter.html

```diff
@@ -1,3 +1,3 @@
 <div>
-    User ID or <em>external</em> group name of person or people permitted to respond to the input.
+    User IDs and/or <em>external</em> group names of person or people permitted to respond to the input, splitted by ','.
```

Comment:
Best to clarify that whitespace is not trimmed: `alice, bob` will not match `alice`.